backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/label/hazard_label.jrxml

```diff
@@ -0,0 +1,74 @@
+	<property name="com.jaspersoft.studio.data.sql.tables" value=""/>
+	<property name="com.jaspersoft.studio.unit." value="pixel"/>
+	<property name="com.jaspersoft.studio.report.description" value=""/>
+	<property name="com.jaspersoft.studio.unit.pageHeight" value="pixel"/>
+	<property name="com.jaspersoft.studio.unit.pageWidth" value="pixel"/>
+	<property name="com.jaspersoft.studio.unit.topMargin" value="pixel"/>
+	<property name="com.jaspersoft.studio.unit.bottomMargin" value="pixel"/>
+	<property name="com.jaspersoft.studio.unit.leftMargin" value="pixel"/>
+	<property name="com.jaspersoft.studio.unit.rightMargin" value="pixel"/>
+	<property name="com.jaspersoft.studio.unit.columnWidth" value="pixel"/>
+	<property name="com.jaspersoft.studio.unit.columnSpacing" value="pixel"/>
+	<import value="net.sf.jasperreports.engine.*"/>
+	<import value="java.util.*"/>
+	<import value="net.sf.jasperreports.engine.data.*"/>
+	<style name="Barcode" forecolor="#000000" fontName="Arial" fontSize="11"/>
+	<parameter name="M_Product_ID" class="java.math.BigDecimal"/>
+	<parameter name="ad_language" class="java.lang.String" isForPrompting="false">
+		<defaultValueExpression><![CDATA[$P{REPORT_LOCALE}.toString()]]></defaultValueExpression>
+	</parameter>
+	<queryString>
+		<![CDATA[SELECT   h.name            AS hazard_name,
+                h.ad_image_id     AS hazard_image_id
+FROM  M_Product_HazardSymbol ph 
+         LEFT JOIN M_HazardSymbol h ON ph.M_HazardSymbol_id = h.M_HazardSymbol_id
+ WHERE ph.m_product_id = $P{M_Product_ID};]]>
+	</queryString>
+	<field name="hazard_name" class="java.lang.String">
+		<property name="com.jaspersoft.studio.field.label" value="hazard_name"/>
+	</field>
+	<field name="hazard_image_id" class="java.math.BigDecimal">
+		<property name="com.jaspersoft.studio.field.label" value="hazard_image_id"/>
+	</field>
+	<background>
+		<band splitType="Stretch"/>
+	</background>
+	<pageHeader>
+		<band splitType="Stretch">
+			<printWhenExpression><![CDATA[( $V{PAGE_NUMBER}.intValue() > 1 ? new Boolean(true) : new Boolean(false))]]></printWhenExpression>
+		</band>
+	</pageHeader>
+	<columnHeader>
+		<band splitType="Stretch"/>
+	</columnHeader>
+	<detail>
+		<band height="38" splitType="Stretch">
+			<property name="com.jaspersoft.studio.unit.height" value="pixel"/>
+			<image scaleImage="FillFrame" hAlign="Center" onErrorType="Blank">
+				<reportElement key="image-1" x="0" y="4" width="30" height="30" uuid="314ff598-49d7-4b36-9eb5-9e8dca503f00"/>
+				<box>
+					<topPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<leftPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<bottomPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<rightPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+				</box>
+				<imageExpression><![CDATA["de/metas/docs/label/AD_Image-"+$F{hazard_image_id}+".jpg"]]></imageExpression>
```

Comment:
make sure does not fail in case hazard_image_id is null or ZERO or some missing ID